lib/features/scanner/telos_signing_manager.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:dart_esr/dart_esr.dart';
+
+class SeedsESR {
+
+  SigningRequestManager manager;
+
+  List<Action> actions;
+
+  SeedsESR({String uri}) {
+    manager = TelosSigningManager.from(uri);
+  }
+
+  Future<void> resolve({String account}) async {
+    this.actions = await manager.fetchActions(account: account);
+  }
+
+  Action firstAction() {
+    return actions[0];
```

Comment:
nickpick - .first

Reply:
Happy to comply with standards

Reply:
a function called "first" seems weird - first what? This is a convenience accessor to get the first list element in actions, so that's not all over the code, as we currently only process single action transactions. But the object is more like a convenience object to encapsulate functionality.